lib/client.ml

```diff
@@ -141,8 +349,26 @@ let init ~state ?(router = fun m -> m) ~msg_handler (address, port) =
   let socket = Mutex.create socket in
   let state = Mutex.create (ref state) in
   let inbox = Inbox.create () in
+  let failure_detector =
+    Failure_detector.make
+      { protocol_period = 9; round_trip_time = 3; peers_to_ping = 3 } in
+  let peers =
+    Base.Hashtbl.create ~growth_allowed:true ~size:0 (module Address) in
+  init_peers
+  |> List.map (fun addr ->
+         Base.Hashtbl.add peers ~key:addr ~data:(Peer.from addr))
+  |> ignore;
```

Comment:
I am not found of it and I do prefer the `let _ =` syntax. Any argument in favor of this one? 😄

Reply:
Nope, we can do let _ = ... in